core/field_textinput.js

```diff
@@ -433,7 +433,7 @@ Blockly.FieldTextInput.oxocardNameValidator = function(text) {
 		return null;
 	}
 	text = String(text);
-	var pattern = new RegExp("[^a-zA-Z0-9 ,;]+", "g");	// match everything except the white listed characters
+	var pattern = new RegExp("[^a-zA-Z0-9 ,;*]+", "g");	// match everything except the white listed characters
```

Comment:
doesn't this need to be escaped?

https://stackoverflow.com/questions/15205536/how-to-escape-asterisk-in-regexp

Reply:
Doesn't seem to be necessary within the brackets [].
It working locally: All characters are frobidden except these whitelisted ones.